### ISSUE TITLE
While using PHP my admin 5.0.4 with My SQL version 8.0.22 . In the exports section I am getting such errors

### ISSUE THREAD

**Issue by kingx619** [phpmyadmin/phpmyadmin#16515]
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#1998
 Required parameter $currentfont follows optional parameter $isunicode

Backtrace
```
tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#2024
 Required parameter $currentfont follows optional parameter $isunicode

Backtrace

tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#2040
 Required parameter $currentfont follows optional parameter $setbom

Backtrace

tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#2060
 Required parameter $currentfont follows optional parameter $setbom

Backtrace

tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#2077
 Required parameter $currentfont follows optional parameter $str

Backtrace

tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_fonts.php#2093
 Required parameter $currentfont follows optional parameter $str

Backtrace

tcpdf.php#115: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_static.php#1139
 Required parameter $tagvspaces follows optional parameter $default_css

Backtrace

tcpdf.php#121: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
Deprecation Notice in .\vendor\tecnickcom\tcpdf\include\tcpdf_static.php#2510
 Required parameter $k follows optional parameter $points

Backtrace

tcpdf.php#121: require_once()
ClassLoader.php#444: include(.\vendor\tecnickcom\tcpdf\tcpdf.php)
ClassLoader.php#322: Composer\Autoload\includeFile(string 'C:\\laragon\\etc\\apps\\phpMyAdmin\\vendor\\composer/../tecnickcom/tcpdf/tcpdf.php')
Composer\Autoload\ClassLoader->loadClass(string 'TCPDF')
ExportPdf.php#25: class_exists(string 'TCPDF')
Plugins.php#107: include_once(.\libraries\classes\Plugins\Export\ExportPdf.php)
Export.php#692: PhpMyAdmin\Plugins::getPlugins(
string 'export',
string 'libraries/classes/Plugins/Export/',
array,
)
server_export.php#49: PhpMyAdmin\Display\Export->getDisplay(
string 'server',
string '',
string '',
string '',
integer 0,
integer 0,
string '<div> <p> <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', true); return false;"> Select all </a> / <a href="#" onclick="Functions.setSelectOptions(\'dump\', \'db_select[]\', false); return false;"> Unselect all </a> </p> <select name="db_select[]" id="db_select" size="10" multiple> <option value="mvc_framework" selected> mvc_framework </option> <option value="phpmyadmin" selected> phpmyadmin </option> </select> </div> ',
)
```

**Comment by ibennetch** [comment]
Hi, could you please also tell us which PHP version you are using?

**Comment by kingx619** [comment]
I am using PHP version 8.0

**Comment by williamdes** [comment]
Duplicate of #16268

**Comment by williamdes** [comment]
Hi @kingx619 
This issue is known and we are waiting for tcpdf (cc @nicolaasuni) to accept the fix for PHP 8.0
